sway/criteria.c

```diff
@@ -144,8 +158,72 @@ static void find_urgent_iterator(struct sway_container *con, void *data) {
 	list_add(urgent_views, con->view);
 }
 
+static bool criteria_matches_container(struct criteria *criteria,
```

Comment:
It looks like i3 only matches containers by `con_id` and `con_mark`. The rest of the options will only match windows/views.

https://github.com/i3/i3/blob/fc01ece3657e334d9067c40c0e7274ee955345e5/src/commands.c#L190

Reply:
It also looks like i3 ignores any other terms when the container has no window. I verified this by doing `i3-msg '[con_mark="test" class="do_not_match"] focus` and it still focused the container. I find this inconsistent, but I'm happy to do the same for compatibility.